usr/share/rear/lib/_input-output-functions.sh

```diff
@@ -791,6 +791,34 @@ preferably the whole debug information via 'rear -D $WORKFLOW'
 ===================="
 }
 
+# Error out with a deprecation info
+# first arg is feature keyword
+# everything else is a reason for the deprecation
+function ErrorIfDeprecated () {
+    local feature="$1" ; shift
```

Comment:
From the current code of the ErrorIfDeprecated function
it seems it must be called at least with a feature argument
so something like
```
test "$feature" || BugError "ErrorIfDeprecated function called without feature argument"
```
should be added to make it more fail-safe.